x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameTransformTask.java

```diff
@@ -553,12 +553,15 @@ protected void doSaveState(IndexerState indexerState, Map<String, Object> positi
                                     },
                                     statsExc -> {
                                         logger.error("Updating stats of transform [" + transformConfig.getId() + "] failed", statsExc);
+                                        auditor.warning(getJobId(),
+                                            "Failure updating persisting stats of transform: " + statsExc.getMessage());
```

Comment:
Maybe remove the word "persisting", i.e. `updating persisting stats` -> `updating stats`?